recipes/force-delete-win/THIRDPARTY.yml

```diff
@@ -0,0 +1,236 @@
+---
+root_name: force-delete-win-py
```

Comment:
As the file is now generated as part of the build, can you delete this static one here?